sentences/pl/media_player_HassSetVolume.yaml

```diff
@@ -0,0 +1,8 @@
+language: pl
+intents:
+  HassSetVolume:
+    data:
+      - sentences:
+          - "<numeric_value_set> [poziom głośności|głośność] <name> na <volume>"
```

Comment:
In my opinion, all below sentences are wrong. 
- "Ustaw poziom głośności telewizora 50%"
- "Zmień głośność telewizora 50%"
- "Przestaw poziom głośności telewizora 50 procent"
- "Przestaw głośność telewizora 50 procent"
- "Nastaw głośność telewizora 50%"
- "Ustaw telewizora 50%"
- "Zmień telewizora 50%"
- "Przestaw telewizor 50 procent"
- "Nastaw telewizor 50%"
- "Zmniejsz głośność telewizora 50%"
- "Zwiększ głośność telewizora 50%"
Please take a look at the tests and if we need to extend them, we can do so. https://github.com/witold-gren/intents/blob/main/tests/pl/media_player_HassSetVolume.yaml#L3

Reply:
In general I agree but in that case why are we supporting something like `- "[<turn_on>|graj] (następn(a|ą|y)|kolejn(y|ą)) [piosenk(a|ę)|utwór|element|ścieżk(a|e|ę)] [w|na] <name>"` and it also generates sentences like
`graj następna piesenka google nest`. It's quite a similiar case, isn't it?

Reply:
I updated sentences when we need play a next song.  Now we have several correct sentences instead of one complicated sentence. This solution will cause that there will be no wrong sentences. I also added preposition as a mandatory to all media player sentences.

Reply:
Cool, looks good.